vault/core.go

```diff
@@ -1426,28 +1432,28 @@ func (c *Core) preSeal() error {
 		c.metricsCh = nil
 	}
 	if err := c.teardownAudits(); err != nil {
-		return err
+		retErr = errwrap.Wrapf("[ERR] error tearing down audits: {{err}}", err)
```

Comment:
and actually, if you want to run all of them, we maybe should use hashicorp/go-multierror  to accumulate errors.